### ISSUE TITLE
Error: Cannot find module './utils/regEx'

### ISSUE THREAD

**Issue by GitHubPang** [naddison36/sol2uml#77]
Seems that the `utils` subfolder is missing from the npm package? 🤔

```
$ yarn init -y
$ yarn add sol2uml
$ yarn sol2uml --help
yarn run v1.22.18
$ /home/user/node_modules/.bin/sol2uml --help
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module './utils/regEx'
Require stack:
- /home/user/node_modules/sol2uml/lib/converterClass2Dot.js
- /home/user/node_modules/sol2uml/lib/converterClasses2Dot.js
- /home/user/node_modules/sol2uml/lib/sol2uml.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/user/node_modules/sol2uml/lib/converterClass2Dot.js:6:17)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/user/node_modules/sol2uml/lib/converterClass2Dot.js',
    '/home/user/node_modules/sol2uml/lib/converterClasses2Dot.js',
    '/home/user/node_modules/sol2uml/lib/sol2uml.js'
  ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

```
$ ls -la node_modules/sol2uml/lib/
total 155
drwxr-xr-x 1 user 1049089     0 Jul 25 13:17 ./
drwxr-xr-x 1 user 1049089     0 Jul 25 13:17 ../
-rw-r--r-- 1 user 1049089   184 Jul 25 12:48 associations.d.ts
-rw-r--r-- 1 user 1049089  1608 Jul 25 12:48 associations.js
-rw-r--r-- 1 user 1049089   227 Jul 25 12:48 converterAST2Classes.d.ts
-rw-r--r-- 1 user 1049089 22567 Jul 25 12:48 converterAST2Classes.js
-rw-r--r-- 1 user 1049089   422 Jul 25 12:48 converterClass2Dot.d.ts
-rw-r--r-- 1 user 1049089  8618 Jul 25 12:48 converterClass2Dot.js
-rw-r--r-- 1 user 1049089   333 Jul 25 12:48 converterClasses2Dot.d.ts
-rw-r--r-- 1 user 1049089  5246 Jul 25 12:48 converterClasses2Dot.js
-rw-r--r-- 1 user 1049089   990 Jul 25 12:48 converterClasses2Storage.d.ts
-rw-r--r-- 1 user 1049089 14976 Jul 25 12:48 converterClasses2Storage.js
-rw-r--r-- 1 user 1049089   253 Jul 25 12:48 converterStorage2Dot.d.ts
-rw-r--r-- 1 user 1049089  3860 Jul 25 12:48 converterStorage2Dot.js
-rw-r--r-- 1 user 1049089   423 Jul 25 12:48 filterClasses.d.ts
-rw-r--r-- 1 user 1049089  2364 Jul 25 12:48 filterClasses.js
-rw-r--r-- 1 user 1049089   350 Jul 25 12:48 index.d.ts
-rw-r--r-- 1 user 1049089  1320 Jul 25 12:48 index.js
-rw-r--r-- 1 user 1049089  1780 Jul 25 12:48 parserEtherscan.d.ts
-rw-r--r-- 1 user 1049089  7122 Jul 25 12:48 parserEtherscan.js
-rw-r--r-- 1 user 1049089   647 Jul 25 12:48 parserFiles.d.ts
-rw-r--r-- 1 user 1049089  4541 Jul 25 12:48 parserFiles.js
-rw-r--r-- 1 user 1049089   192 Jul 25 12:48 parserGeneral.d.ts
-rw-r--r-- 1 user 1049089  1538 Jul 25 12:48 parserGeneral.js
-rwxr-xr-x 1 user 1049089    32 Jul 25 12:48 sol2uml.d.ts*
-rwxr-xr-x 1 user 1049089  7724 Jul 25 12:48 sol2uml.js*
-rw-r--r-- 1 user 1049089   866 Jul 25 12:48 typeGuards.d.ts
-rw-r--r-- 1 user 1049089  1338 Jul 25 12:48 typeGuards.js
-rw-r--r-- 1 user 1049089  2373 Jul 25 12:48 umlClass.d.ts
-rw-r--r-- 1 user 1049089  4336 Jul 25 12:48 umlClass.js
-rw-r--r-- 1 user 1049089   614 Jul 25 12:48 writerFiles.d.ts
-rw-r--r-- 1 user 1049089  5527 Jul 25 12:48 writerFiles.js
```

**Comment by naddison36** [comment]
thanks for reporting. I'll investigate

**Comment by naddison36** [comment]
This should be fixed in v2.0.2
